rewrite-javascript/build.gradle.kts

```diff
@@ -157,54 +157,37 @@ val npmInstallTemp = tasks.register<NpmTask>("npmInstallTemp") {
     val tempInstallDir = layout.buildDirectory.file("tmp/npmInstall")
     inputs.files(npmPack)
         .withPathSensitivity(PathSensitivity.RELATIVE)
+    outputs.dir(tempInstallDir)
     dependsOn(npmInitTemp)
 
     // Use a provider to defer evaluation until execution time
     args = provider { listOf("install", npmPack.get().archiveFile.get().asFile.absolutePath, "--omit=dev") }
     workingDir.set(tempInstallDir)
 }
 
-sourceSets {
-    main {
-        resources {
-            srcDir("src/main/generated-resources")
```

Comment:
callout: Should always avoid writing to the project directory as it can result in triggering tasks to run unnecessarily.